sparse/_io.py

```diff
@@ -49,11 +50,17 @@ def save_npz(filename, matrix, compressed=True):
 
     nodes = {
         "data": matrix.data,
-        "coords": matrix.coords,
         "shape": matrix.shape,
         "fill_value": matrix.fill_value,
     }
 
+    if type(matrix) == COO:
```

Comment:
Change docstring to say:

1. That it returns a `sparse.SparseArray`
2. That the format isn't stable at this time.

Reply:
To clarify, I'm noting that the GCXS format isn't stable at this time? Or that the format of the of the output array isn't stable? Thanks for your patience, Hameer.

Reply:
My mistake, I should've been more clear: I mean that the output binary format of `save_npz`/`load_npz` isn't stable at this time.

Reply:
I see. Thanks for clarifying.